Mathlib/Algebra/Lie/Basic.lean

```diff
@@ -296,18 +304,16 @@ def LieRing.toNonUnitalNonAssocRing : NonUnitalNonAssocRing L :=
 
 variable {ι κ : Type*}
 
-theorem sum_lie (s : Finset ι) (f : ι → L) (a : L) : ⁅∑ i ∈ s, f i, a⁆ = ∑ i ∈ s, ⁅f i, a⁆ :=
-  let _i := LieRing.toNonUnitalNonAssocRing L
-  s.sum_mul f a
+theorem sum_lie (s : Finset ι) (f : ι → L) (m : M) : ⁅∑ i ∈ s, f i, m⁆ = ∑ i ∈ s, ⁅f i, m⁆ :=
+  let g : L →+ M := ⟨⟨fun x ↦ ⁅x, m⁆, zero_lie m⟩, fun x y ↦ add_lie x y m⟩
+  map_sum g f s
```

Comment:
```suggestion
  map_sum ((LieModule.toEndAddHom L M).flip m) f s
```
probably

Reply:
Oh my previous commit message was a lie: we have `MonoidHom.flip`.